tensorflow_quantum/python/util_test.py

```diff
@@ -61,8 +61,10 @@ class UtilFunctionsTest(tf.test.TestCase, parameterized.TestCase):
     def test_get_supported_gates(self):
         """Confirm one of every gate is returned."""
         mapping_1 = util.get_supported_gates()
-        self.assertEqual(len(mapping_1.keys()),
-                         len(serializer.SERIALIZER.supported_gate_types()))
+        self.assertEqual(
+            len(mapping_1.keys()),
+            len(serializer.SERIALIZER.supported_gate_types()) -
+            len(util._SUPPORTED_CHANNELS))
```

Comment:
Just curious, it may be helpful to add `util.get_supported_channels()`, and use `len(util.get_supported_channels())` rather than the private `len(util._SUPPORTED_CHANNELS)`? WDYT?

Reply:
Good call. Added!